### DISCUSSION TITLE
End of life support date 7.22.x

### DISCUSSION THREAD

**Discussion by sidmishfpl1801** [babel/babel#16361] [Q&A] [answered]
Hello Community,
We are in the process of applying for FDA certification  and need to include information about the end-of-life dates for a packages. Could someone please inform me if there are any available details regarding the end of support for babel version 7.22.x?

**Comment by liuxingbaoyu** [comment, marked as answer]
https://github.com/babel/babel/blob/main/SECURITY.md
Did this help you?

**Reply by sidmishfpl1801** [reply]
@liuxingbaoyu, Thank you for your response. 
According to the SECURITY.md document, if the most current version is 7.23.x, does that mean security updates are not provided for versions 7.22.x or earlier? Additionally, I have included some information; could you verify if it aligns with the document's details![image](https://github.com/babel/babel/assets/89624855/08e0b640-50bf-4dee-aa79-fb6b51b24c2c)

Note that release-date for v7.24.0 was 28-02-2024.

**Reply by nicolo-ribaudo** [reply]
> does that mean security updates are not provided for versions 7.22.x or earlier?

Yes, exactly. We follow semver, so minor releases do not have breaking changes and thus you can safely update to the latest one to get the security fixes it includes.

**Reply by nicolo-ribaudo** [reply]
I'm not sure I understand your table -- 7.23.2 stops being maintained on the same day that it is released, as any fix will go in a release with a higher number.

I think the underlying assumption is wrong for this project: due to how we number releases, you should track the "7" release line and not "7.22", "7.23", "7.24" as separate release lines.

**Reply by sidmishfpl1801** [reply]
@nicolo-ribaudo Thanks for the clarification, i should consider the end of support for 7.x as indicated in the image below.

![image](https://github.com/babel/babel/assets/89624855/e6dc9420-64c1-4a6b-8c28-52c4e67f1b1b)

**Reply by nicolo-ribaudo** [reply]
Yes! As written there there is no an actual "end of support" date yet, given that we don't know when Babel 8 will be released.

What you could do is use "18-03-2025" (in one year) since Babel 7 will be for sure maintained until then (if we were to release Babel 8 tomorrow), and every few months update it again to be "one year from today".

**Reply by sidmishfpl1801** [reply]
Thanks a lot @nicolo-ribaudo for your quick reply, really appreciate that.
